packages/video_player/video_player/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 * Updates support matrix in README to indicate that iOS 11 is no longer supported.
 * Clients on versions of Flutter that still support iOS 11 can continue to use this
   package with iOS 11, but will not receive any further updates to the iOS implementation.
+* Web, Fix infinite pause loop caused by seekTo marking the video as completed when already completed.
```

Comment:
Please, revert all changes to the `video_player` package, they're not needed.

`flutter pub get` (or `flutter pub upgrade`) will take care of fetching the fixed version of `video_player_web` once the fix is published.

(Sorry I wasn't clearer before!)